src/integrationtest/org/apache/poi/stress/HeapDump.java

```diff
@@ -42,7 +43,17 @@ Licensed to the Apache Software Foundation (ASF) under one or more
     public static void dumpHeap(String fileName, boolean live) throws IOException {
         // initialize hotspot diagnostic MBean
         initHotspotMBean();
-        hotspotMBean.dumpHeap(fileName, live);
+        throw new NotImplementedException("Developer: You need to uncomment the corresponding line here.");
```

Comment:
I'd prefer if this was modified to use Java reflection

Reply:
Done! @pjfanning mind taking another look? I tested w/ OpenJ9 11 and OpenJDK 16